org.eevolution.warehouse/src/main/java/base/org/spin/form/OutBoundOrder.java

```diff
@@ -1114,12 +1343,12 @@ public String validStock(IMiniTable stockTable) {
 			//	Get Values
 			String product 				= ((KeyNamePair) stockTable.getValueAt(i, SW_PRODUCT)).getName();
 			String warehouse 			= ((KeyNamePair) stockTable.getValueAt(i, SW_WAREHOUSE)).getName();
-			BigDecimal m_QtyOnHand 		= ((BigDecimal) stockTable.getValueAt(i, SW_QTY_ON_HAND));
-			BigDecimal m_QtyInTransit 	= ((BigDecimal) stockTable.getValueAt(i, SW_QTY_IN_TRANSIT));
-			BigDecimal m_QtySet 		= ((BigDecimal) stockTable.getValueAt(i, SW_QTY_SET));
-			BigDecimal m_QtyAvailable 	= ((BigDecimal) stockTable.getValueAt(i, SW_QTY_AVAILABLE));
+			BigDecimal qtyOnHand 		= ((BigDecimal) stockTable.getValueAt(i, SW_QTY_ON_HAND));
```

Comment:
Please make this code more atomic, the refactory can be change in other pull request as Source code Improve